client/components/domains/search-filters/style.scss

```diff
@@ -118,6 +121,53 @@
 			padding-top: 0.5em;
 			padding-bottom: 0.5em;
 		}
+
+		button.search-filters__popover-button-domain-step-test {
+			flex: 1 0 auto;
+			margin-left: 2em;
+
+			@include breakpoint( '<480px' ) {
+				margin-left: 0;
+			}
+		}
+
+		legend.search-filters__filter-by {
+			font-size: 14px;
+			color: var( --color-text-subtle );
+
+			@include breakpoint( '<480px' ) {
+				display: none;
+			}
+		}
+	}
+
+	.search-filters__tld-checkbox-filter-bar {
+		display: flex;
+		flex: 1 0 auto;
+		justify-content: space-around;
+		font-size: 14px;
+
+		@include breakpoint( '<480px' ) {
+			flex-grow: 0;
+		}
+
+		& .search-filters__tld-checkbox {
```

Comment:
Shorthand (without `&`) will also work! :)

Not important though.

Reply:
Thanks, implemented in [517527c](https://github.com/Automattic/wp-calypso/pull/39346/commits/517527c44eb4b7dd43fbf995cfa270113dc23db1).